backend/package.json

```diff
@@ -45,53 +45,53 @@
     "jsonwebtoken": "^9.0.2",
     "lodash-es": "^4.17.21",
     "mjml": "^4.15.3",
-    "mongodb": "^6.4.0",
+    "mongodb": "^6.5.0",
     "mongoose": "^6.11.3",
     "mongoose-delete": "^0.5.4",
     "morgan": "^1.10.0",
-    "nanoid": "^5.0.6",
+    "nanoid": "^5.0.7",
     "node-fetch": "^3.3.2",
-    "nodemailer": "^6.9.8",
-    "qs": "^6.12.0",
+    "nodemailer": "^6.9.13",
+    "qs": "^6.12.1",
     "semver": "^7.6.0",
     "shelljs": "^0.8.5",
     "utility-types": "^3.11.0",
     "uuid": "^9.0.1",
-    "yargs": "^17.7.1",
-    "zod": "^3.22.4",
+    "yargs": "^17.7.2",
+    "zod": "^3.22.5",
     "zod-error": "^1.5.0"
   },
   "devDependencies": {
-    "@anatine/zod-mock": "^3.13.3",
-    "@faker-js/faker": "^8.0.2",
```

Comment:
Could we update this to be a fixed version? Using different versions have caused our unit tests to fail